### PR TITLE
Analyticstack related changes to support hawthorn

### PR DESCRIPTION
**Pull request by HassanJaveed84**
boxcutter/ubuntu1604 is not available anymore.

### CODE REVIEW

**Comment by HassanJaveed84** [comment]
Going to build the box with the newer HIVE version and test again.

**Comment by HassanJaveed84** [comment]
This is ready for final review.

**Review by azafty468 — APPROVED** [review]
👍 LGTM!

File: vagrant/release/analyticstack/Vagrantfile [hunk 1]
```diff
@@ -48,6 +48,8 @@ end
 # to a name and a file path, which are used for retrieving
 # a Vagrant box from the internet.
 openedx_releases = {
+  "open-release/hawthorn.master" => "master-analyticstack-2018-06-11",
+
```
Comment (brianhw) [inline vagrant/release/analyticstack/Vagrantfile:52]: That looks good enough for now for Hawthorn.  But I'm surprised that there is no entry for gingko.   It should probably be whatever is the current equivalent of the current analyticstack-latest, which we're about to overwrite.  I think there should be a 2017-06-28 image.  Can you remember where to look, or how?  I remember not being able to write there, but can't remember about read access.

Comment (brianhw) [reply vagrant/release/analyticstack/Vagrantfile:52]: Yes, there is a master-analyticstack-2017-06-28.box, and that should be used for gingko.master.  If it's not already uploaded, it should be added to the request, and added here.

Comment (HassanJaveed84) [reply vagrant/release/analyticstack/Vagrantfile:52]: Thanks Brian!
I don't see it in the bucket, I've added this file to the DevOps request. I'll add the mapping once it is copied over.

**Review by macdiesel — APPROVED** [review]

**Review by rao-abdul-mannan — APPROVED** [review]
LGTM 👍

**Review by brianhw — APPROVED** [review]
LGTM.   👍

**Comment by HassanJaveed84** [comment]
@feanil can you please review this ?

**Comment by HassanJaveed84** [comment]
@jibsheet if you get some time Today, I'll appreciate a review so I can merge.

**Comment by jibsheet** [comment]
@HassanJaveed84 this change looks good, go ahead and merge

**Pull request merged**